packages/react-components-lab/src/components/CollapsableBox/CollapsableBox.tsx

```diff
@@ -11,9 +11,29 @@ const CollapsableBox: FC<CollapsableBoxProps> = ({
   className = '',
 }) => {
   const [isCollapsed, setIsCollapsed] = useState(true);
+  const [gradient, setGradient] = useState(
+    'linear-gradient(180deg, rgba(255,255,255,0) 0%, rgba(255,255,255,1) 100%)'
+  );
+
+  const theme = useTheme();
+  useEffect(() => {
+    if (theme.palette.mode === 'light') {
+      setGradient(
+        'linear-gradient(180deg, rgba(255,255,255,0) 0%, rgba(255,255,255,1) 100%)'
+      );
+    } else {
+      setGradient(
+        'linear-gradient(180deg, rgba(0, 0, 0, 0) 0%, rgba(0, 0, 0, 1) 100%)'
+      );
+    }
+  }, [theme.palette.mode]);
```

Comment:
```suggestion
  const theme = useTheme();
  const gradient = theme.palette.mode === 'light' ?  'linear-gradient(180deg, rgba(255,255,255,0) 0%, rgba(255,255,255,1) 100%)' :  'linear-gradient(180deg, rgba(0, 0, 0, 0) 0%, rgba(0, 0, 0, 1) 100%)'

```

Having a useEffect is redundant and does not provide the ideal functionality. Since the variable is dependant on the useTheme value, the component will re-render once the upper provider re-renders, hence, it will simply take the value for the current active theme mode.

Reply:
Please pull and fix the spacing, probably there are small linting issues. Then, update the version of the @dhi/react-components-lab to x.x.^x. Once that is done, push on the branch.

Afterward, locally on your pc run yarn build in the root of react-components-lab. Once you run build, you should then be able to run `npm publish`. If you'd like to do this together, let me know 👍🏼